docs/_ldio/examples/ex3-enrich-model.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: default
+parent: Examples
+title: Enrich a model
+nav_order: 3
+has_toc: true
+---
+
+# Enrich A Model
+
+As part of this example, we will store some Car data in a Graph Database. 
+We will later use that data to extend our user data Model to include the Car data.
+
+## Used Components
+
+- [Http In](../ldio-inputs/ldio-http-in)
+- [RDF Adapter](../ldio-adapters/ldio-rdf-adapter.md)
+- [Console Out](../ldio-outputs/ldio-console-out)
+- [Repository Materialiser](../ldio-outputs/ldio-repository-materialiser)
+
+## Setup
+
+For this setup, we will have two pipelines:
```

Comment:
It's a pity we can't use output components in the middle of the pipeline ;)

Reply:
not happening ;)